t/admin/plugins.t

```diff
@@ -166,9 +166,27 @@ plugins:
 
 
 === TEST 8: get all the attributes of all plugins
```

Comment:
add a test to confirm the plugin of `auth` type

Reply:
add